syntaxes/injection.json

```diff
@@ -0,0 +1,17 @@
+{
+    "scopeName": "memo.wikilink.injection",
+    "injectionSelector": "L:meta.paragraph.markdown",
```

Comment:
probably would make sense to apply it also to headers

Reply:
Ok, I will take a stab at this later today.

Reply:
Please check my last commit https://github.com/svsool/memo/pull/561/commits/21d25609ec0b51706061891e42c0b2c89daaac5d.